go/private/context.bzl

```diff
@@ -357,17 +357,19 @@ def _check_importpaths(ctx):
         if ":" in p:
             fail("import path '%s' contains invalid character :" % p)
 
-def _infer_importpath(ctx):
+def _infer_importpath(ctx, attr = None):
```

Comment:
I think that this is only called in one place and that call site provides `attr`, so we can make this attribute required.

Reply:
Good idea. Done.